src/error-request.ts

```diff
@@ -10,7 +10,19 @@ export async function errorRequest(state, octokit, error, options) {
   if (error.status >= 400 && !state.doNotRetry.includes(error.status)) {
     const retries =
       options.request.retries != null ? options.request.retries : state.retries;
-    const retryAfter = Math.pow((options.request.retryCount || 0) + 1, 2);
+    const base = (options.request.retryCount || 0) + 1;
+    let retryAfter;
+    switch (state.strategy) {
+      case "exponential":
+        retryAfter = Math.pow(2, base);
+        break;
+      case "linear":
+        retryAfter = base;
+        break;
+      default: // "polynomial"
+        retryAfter = Math.pow(base, 2);
```

Comment:
For the polynomial strategy, you'll want `Math.pow(2, base)`

Also, you might want to wrap this in `Math.round(Math.pow(base, exp))` just because `pow` returns a double

Reply:
Wrapped `Math.pow` with `Math.round` - great spot, thanks 😄 

As mentioned in https://github.com/octokit/plugin-retry.js/pull/412#discussion_r1133750985, I think that would make it exponential backoff. 

Full disclosure, I named this one polynomial by looking at what others tool call it (e.g. https://backoff-utils.readthedocs.io/en/latest/strategies.html#polynomial).